eng/testing/xunit/xunit.console.targets

```diff
@@ -85,13 +92,6 @@
             CopyToOutputDirectory="PreserveNewest"
             Visible="false" />
     </ItemGroup>
-
-    <!-- Workaround for https://github.com/xunit/xunit/issues/1651 -->
-    <ItemGroup Condition="'$(ArchiveTests)' != 'true'">
-      <None Remove="$(Pkgxunit_runner_visualstudio)\build\net452\xunit.runner.utility.net452.dll" />
-      <None Remove="$(Pkgxunit_runner_visualstudio)\build\net452\xunit.runner.reporters.net452.dll" />
-      <None Remove="$(Pkgxunit_runner_visualstudio)\build\netcoreapp2.1\xunit.runner.utility.netcoreapp10.dll" />
-      <None Remove="$(Pkgxunit_runner_visualstudio)\build\netcoreapp2.1\xunit.runner.reporters.netcoreapp10.dll" />
-    </ItemGroup>
```

Comment:
Unrelated change: This isn't necessary anymore as the underlying bug was fixed in xunit.